packages/babel-plugin-proposal-class-properties/test/fixtures/private/tagged-template/input.js

```diff
@@ -0,0 +1,8 @@
+class Foo {
+    #tag() {
+      return this;
+    }
+    
+    #tag2 = this.#tag;
+  }
+  new Foo();
```

Comment:
Here we are not testing tagged templates, could you rewrite it to something like this?

```js
class Foo {
  #tag;

  test() {
    this.#tag``;
  }
}
```